common-arcana.lic

```diff
@@ -133,6 +142,29 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def prepare_to_cast_runestone(spell, settings)
+    if DRCI.inside?("#{spell['runestone_name']}", settings.runestone_storage)
+      return false if !get_runestone(spell['runestone_name'], settings)
+    else
+      DRC.message("*** Out of #{spell['runestone_name']}! ***")
+      return false
+    end
+    DRC.bput("harness #{settings.runestone_harness}", "You tap into the mana") if settings.runestone_harness
+    waitcastrt?
+    return true
+  end
+
+  def get_runestone(runestone, settings)
+    #return if DRC.left_hand =~ /runestone/ || DRC.right_hand =~ /runestone/
+    return if DRCI.in_hands?('runestone')
```

Comment:
Change from the literal string `'runestone'` to the variable `runestone`
```suggestion
    return if DRCI.in_hands?(runestone)
```

Reply:
Changed.